docs/schemas/AgInspectionReport.json

```diff
@@ -0,0 +1,72 @@
+{
+    "$id": "https://w3id.org/traceability/schemas/AgInspectionReport.json",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$comment": "{\"term\": \"AgInspectionReport\", \"@id\": \"https://w3id.org/traceability/AgInspectionReport\"}",
+    "title": "Agriculture Inspection Report",
+    "description": "Information on the Inspection and the observations made",
+    "type": "object",
+    "properties": {
+      "@context": {
+        "type": "array"
+      },
+      "type": {
+          "oneOf": [
+              { "type": "string" },
+              { "type": "array"}
+          ]
+      },
+      "facilityID": {
```

Comment:
wondering if this should also be a `facility` object of type `Organization` or similar, that might have an `id` property.

Reply:
ping @msporny  @dlongley  any thoughts on this structure?

Reply:
Given that an organization can have multiple facility locations we could have a location property in there?

Reply:
I think an location property would be fine, but there will be a distinct id (alpha-num) assigned to that facility that we need to record as well

Reply:
> wondering if this should also be a facility object of type Organization or similar, that might have an id property.

Agreed, especially if we're going to use something like a GLN to identify a facility (I hope so?). Ideally, we identify facilities using URLs so that we can add more linked data to that facility over time. Keeping these as non-URL identifiers won't help us enrich the data in the future.